src/flb_input_chunk.c

```diff
@@ -1301,7 +1301,7 @@ int flb_input_chunk_append_raw(struct flb_input_instance *in,
      * Keep the previous real size just to satisfy
      * flb_input_chunk_update_output_instances().
      */
-    pre_real_size = flb_input_chunk_get_real_size(ic);
+    pre_real_size = new_chunk == FLB_TRUE ? 0 : flb_input_chunk_get_real_size(ic);
```

Comment:
please use a normal ```if``` statement to help readability

Reply:
done